client/signup/steps/design-picker/index.jsx

```diff
@@ -75,6 +77,7 @@ class DesignPickerStep extends Component {
 				stepContent={ this.renderDesignPicker() }
 				align={ isReskinned ? 'left' : 'center' }
 				skipButtonAlign={ isReskinned ? 'top-right' : 'bottom' }
+				className={ largeThumbnails && 'design-picker-step__is-large-thumbnails' }
```

Comment:
Maybe adding the class to `DesignPicker` is better

Reply:
Perhaps you're right. I was trying to not change the existing components too much, but I think I should just add a `className` prop to `<DesignPicker>`.